variables.tf

```diff
@@ -216,6 +222,58 @@ variable "kubernetes_version" {
   default     = null
 }
 
+variable "load_balancer_profile_idle_timeout_in_minutes" {
+  type        = number
+  description = "(Optional) Desired outbound flow idle timeout in minutes for the cluster load balancer. Must be between `4` and `120` inclusive."
+  default     = null
+  nullable    = true
+}
+
+variable "load_balancer_profile_managed_outbound_ip_count" {
+  type        = number
+  description = "(Optional) Count of desired managed outbound IPs for the cluster load balancer. Must be between `1` and `100` inclusive"
+  default     = null
+  nullable    = true
```

Comment:
`nullable` is default to `true` so I think we could just omit this expression here.